src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistDetailsPlaceholder.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import {
+  Box,
+  Column,
+  GridColumns,
+  Separator,
+  SkeletonBox,
+  SkeletonText,
+} from "@artsy/palette"
+import { Carousel } from "v2/Components/Carousel"
+
+export interface PartnerArtistDetailsPlaceholderProps {
+  done?: boolean
+}
+
+export const PartnerArtistDetailsPlaceholder: React.FC<PartnerArtistDetailsPlaceholderProps> = ({
+  done = true,
+}) => {
+  return (
+    <Box>
+      <Separator mt={4} />
+      <GridColumns gridRowGap={[2, 4]} my={4}>
+        <Column span={6}>
+          <GridColumns gridRowGap={2}>
+            <Column span={12}>
+              <SkeletonText done={done} variant="largeTitle">
+                Darren Almond
```

Comment:
I'm not sure if these strings actually show up, but can we use placeholders like "Artist Name" etc. instead of real names here?

(Same goes for the fields below)

Reply:
Fixed